frontend/components/TeamSection/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { GradientText } from '../shared/Heading';
+import PersonCard from '../person-card';
+
+const TeamSection = ({ data }) => {
+  const {
+    title,
+    userOne,
+    userTwo,
+    userTitle,
+    description,
+    userOneImg,
+    userTwoImg
+  } = data;
+
+  const userOneData = {
+    image: userOneImg,
+    title: userOne,
+    subtitle: userTitle,
+    description: description
+  };
+  const userTwoData = {
+    image: userTwoImg,
+    title: userTwo,
+    subtitle: userTitle,
+    description: description
+  };
+
+  return (
+    <div className="text-center">
+      <GradientText
```

Comment:
don't use gradient text alone .. use the heading component as described in code example componnet in heading storybook
also don't use the inline block for the gradient text as it will bug the background gradient